.github/workflows/build-and-test.yml

```diff
@@ -69,6 +69,12 @@ jobs:
           exit 1
         fi
 
+    - name: Install Trivy
+      run: |
+        wget https://github.com/aquasecurity/trivy/releases/download/v0.61.1/trivy_0.61.1_Linux-64bit.tar.gz
```

Comment:
Can we sha256 this file and ensure it matches the expected value? This just avoids the file from changing unexpectantly.